src/main/drivers/barometer/barometer_lps.c

```diff
@@ -0,0 +1,166 @@
+#include <stdbool.h>
+#include <stdint.h>
+
+#include <platform.h>
+
+#if defined(BARO) && (defined(USE_BARO_LPS) || defined(USE_BARO_SPI_LPS))
```

Comment:
If only `USE_BARO_SPI_LPS` is defined, the baro won't be initialised. We should probably not check for this define in the current implementation (or rename `USE_BARO_LPS` to `USE_BARO_SPI_LPS`) if we are only supporting SPI for now.

Reply:
Fair enough so we will stick just to SPI option

Reply:
We can always add I2C in later - hence the idea to use `USE_BARO_SPI_LPS` as the name of the define.

Reply:
Why was this not fixed? Which of these defines it to be used now?